pkg/pillar/cmd/usbmanager/subscriptions.go

```diff
@@ -0,0 +1,225 @@
+// Copyright (c) 2023 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+package usbmanager
+
+import (
+	"reflect"
+	"time"
+
+	"github.com/lf-edge/eve/pkg/pillar/hypervisor"
+	"github.com/lf-edge/eve/pkg/pillar/pubsub"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+)
+
+func (usbCtx *usbmanagerContext) process(ps *pubsub.PubSub) {
+	stillRunning := time.NewTicker(stillRunningInterval)
+
+	watches := make([]pubsub.ChannelWatch, 0)
+	for i := range usbCtx.subscriptions {
+		sub := usbCtx.subscriptions[i]
+		watches = append(watches, pubsub.ChannelWatch{
+			Chan: reflect.ValueOf(sub.MsgChan()),
```

Comment:
this `reflect.ValueOf` is not ideal yet; not sure if it can be made nicer with generics